content/en/agent/guide/fips-agent.md

```diff
@@ -113,9 +113,15 @@ The Datadog FIPS Agent is in preview and has not been fully audited. Install and
 {{% /tab %}}
 {{< /tabs >}}
 
+## Post installation
+
+For more information FIPS compliant services on Datadog integrations, see [FIPS Verified Agent Integrations][1].
```

Comment:
```suggestion
For more information FIPS compliant services on Datadog integrations, see [FIPS Verified Agent Integrations][3].
```

Reply:
I recommend moving this to Further Reading (located at the top of the page in frontmatter)